apps/meteor/ee/app/license/server/airGappedRestrictions.ts

```diff
@@ -0,0 +1,44 @@
+import { cronJobs } from '@rocket.chat/cron';
+import { AirGappedRestriction } from '@rocket.chat/license';
+import { Settings } from '@rocket.chat/models';
+import { Meteor } from 'meteor/meteor';
+
+import { notifyOnSettingChangedById } from '../../../../app/lib/server/lib/notifyListener';
+import { settings } from '../../../../app/settings/server';
+import { i18n } from '../../../../server/lib/i18n';
+import { sendMessagesToAdmins } from '../../../../server/lib/sendMessagesToAdmins';
+import { checkAirGappedRestrictions } from './airGappedRestrictionsCheck';
+
+const updateRestrictionSetting = async (remainingDays: number) => {
+	const value = settings.get('Cloud_Workspace_AirGapped_Restrictions_Remaining_Days');
+	if (value === remainingDays) {
+		return;
+	}
+	await Settings.updateValueById('Cloud_Workspace_AirGapped_Restrictions_Remaining_Days', remainingDays);
```

Comment:
Maybe, we can create a new method that does the findOneAndUpdate kind of update and that way we avoid `notifyOnSetting...` to find the updated setting again.

Reply:
I discussed this with @tassoevan yesterday, we concluded that this might not be the best place or time to do this. Specially since this is supposed to run at max twice a day, so shouldn't be a performance issue.